test/nidmExampleDataTest.m

```diff
@@ -1,10 +1,46 @@
+%To run the below tests use the following code:
+%import matlab.unittest.TestSuite;
+%tests = matlab.unittest.TestSuite.fromFile(which('nidmExampleDataTest'))
+%result = run(tests)
+
 classdef nidmExampleDataTest < matlab.unittest.TestCase
+    
+    methods(TestMethodSetup)
+        %Rename the users HTML folder to prevent the tests damaging it.
+        function storeUsersHTML(testCase)
+            if exist(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html')) == 7 
+                movefile(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html'), strrep(fileparts(mfilename('fullpath')), 'test', 'Data\htmlTemp'))
+            end
+        end
+    end
+    
+    methods(TestMethodTeardown)
+        %Remove the HTML folder created by test.
+        function removeTestHTML(testCase)
+            pause(0.1);
+            if exist(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html')) == 7
```

Comment:
You can use `dir` as a second argument to `exist` to only check for directories.